.eslintrc

```diff
@@ -27,5 +27,13 @@
         "object-curly-newline": 0,
         "operator-linebreak": [2, "before"],
         "sort-keys": 0,
-    }
+    },
+    "overrides": [
+        {
+            "files": "test/resolver/nested_symlinks/mylib/*.js",
+            "rules": {
+                "no-throw-literal": 0,
```

Comment:
Instead of ` throw 'async: no match';` you could just do ` throw new Error('async: no match');`

Reply:
sure, but then the failure output includes a stack trace, which makes it harder to read :-)